website/content/api-docs/system/managed-keys.mdx

```diff
@@ -213,6 +213,39 @@ $ curl \
 
 - `key_type` `(string: <required>)`: The type of key to use. At this time only supported value is `RSA`.
 
+#### GCP Cloud KMS Parameters
+
+- `credentials` `(string: <required>)`: The path of the credential file to use for authenticating to GCP.
+This can also be provided in the `GOOGLE_APPLICATION_CREDENTIALS` environment variable.
+
+- `crypto_key` `(string: <required>)`: The name of the GCP Cloud KMS key. If there is no existing key
+and `allow_generate_key` is `true`, Vault will generate a key with this name.
+
+- `crypto_key_version` `(string: "1")`: The version of the key to use.
+
+- `key_ring` `(string: <required>)`: The name of the key ring in GCP Cloud KMS.
+
+- `project` `(string: <required>)`: The ID of the GCP project.
+
+- `region` `(string: <required>)`: The region where the key ring was created. This can also be provided
+with the `GOOGLE_REGION` environment variable.
+
+- `signature_algorithm` `(string: <required)`: The signature algorithm to be used with the key. Supported
```

Comment:
I believe this field is `algorithm` from the managed key point of view, within go-kms-wrapper lib it is mapped to `signature_algorithm`